lerna.json

```diff
@@ -0,0 +1,6 @@
+{
+  "packages": [
+    "packages/*"
```

Comment:
I would like it better if the packages are visible on the root level of the repository without the extra `packages/` directory. If there are more practical benefits then I'd of course be fine with having it, but if this glob here is the only one, then I'd rather list them here one by one.